### PR TITLE
docs : add WeTransfer to 'Who uses Amundsen?'

### PR DESCRIPTION
**Pull request by higee**
### Summary of Changes

Adds WeTransfer to `Who uses Amundsen?` in the README

As a side note, I'm sharing some of the things we did differently. Maybe helpful for some people.

- use Kuberenetes Ingress
- set up authentication with [OAuth Proxy](https://github.com/oauth2-proxy/oauth2-proxy)
- separate [config](https://github.com/amundsen-io/amundsendatabuilder/blob/71492a96066060cbc6395a3a9afdb6289f3ad491/example/dags/postgres_sample_dag.py#L87-L110) and application codes by using templated json (i.e .j2) for configuration and render values when passing them to Airflow tasks 
- use Factory Pattern so that developers with less experience in Amundsen can simply pass job, task, publisher, extractor, loader, transformer and prepare a j2 file.

```
# returns DefaultJob
runner = DataBuilderRunner(
    job_type=dag_args.pop("job"),
    publisher_type=dag_args.pop("publisher"),
    task_type=dag_args.pop("task"),
    extractor_type=dag_args.pop("extractor"),
    transformer_type=dag_args.pop("transformer"),
    loader_type=dag_args.pop("loader"),
    job_config=dag_args.pop("job_config"),
    logger=logger,
    **dag_args
)

# runs DefaultJob.launch() along with other custom actions
runner.execute()
```

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [x] PR title addresses the issue accurately and concisely, including a title prefix.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,6 +215,7 @@ Currently **officially** using Amundsen:
 1. [Remitly](https://www.remitly.com/)
 1. [Square](https://squareup.com/us/en)
 1. [Workday](https://www.workday.com/en-us/homepage.html)
+1. [WeTransfer](https://wetransfer.com)
```
Comment (feng-tao) [inline README.md:218]: thanks @higee ! the list is sorted alphabetically, could you put it before workday?

Comment (higee) [reply README.md:217]: @feng-tao my bad. Pushed a new commit :)

Comment (feng-tao) [reply README.md:217]: no worry, thanks!

**Review by feng-tao — APPROVED** [review]

**Comment by feng-tao** [comment]
also @higee , your note is useful for folks who are not familiar with databuilder and its different configs. Maybe you could share that to the slack channel?

**Pull request merged**